hilti/include/ast/operators/map.h

```diff
@@ -69,13 +69,14 @@ STANDARD_OPERATOR_2x_lhs(map, IndexNonConst, Index, operator_::elementType(0), t
 BEGIN_METHOD(map, Get)
```

Comment:
The change on the C++ side makes sense. However, I'm wondering if we could keep the signature & semantics for the HILTI-side `get()` method the same as they were and handle the default argument during code generation. 
As an example, ``m.get("key", "default")`` could produce some version of `(m.has("key") ? m["key"] : "default")`.